pkg/netpol/connlist/connlist_test.go

```diff
@@ -465,5 +465,5 @@ func TestConnlistAnalyzerIllegalPodList(t *testing.T) {
 	res, _, err := analyzer.ConnlistFromDirPath(dirPath)
 	require.Nil(t, res)
 	require.Contains(t, err.Error(), "Input Pod resources are not supported for connectivity analysis."+
-		" Found Pods of the same owner but different set of labels.")
+		" Found Pods of the same owner demo/cog-agents but with different set of labels.")
```

Comment:
I think we should avoid hard-coded/duplicated error messages for testing. Ideally, we should have a pkg with defined error strings as const values , that will be used both for error generation and for testing errors strings cross-packages. You can open another issue so we consider handling this later.

Reply:
added as a new task in #203